### ISSUE TITLE
[SampleCard-ExpenseReport; Config-Windows-timeline.json]: The links 'ER-13052' and 'Show History' have insufficient luminosity ratio for multiple config files.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7376]
https://microsoft.visualstudio.com/OS/_workitems/edit/37987139
